src/ucp/tag/tag_recv.c

```diff
@@ -103,7 +129,8 @@ ucp_tag_recv_common(ucp_worker_h worker, void *buffer, size_t count,
                                                      req->recv.length);
     req->recv.tag.tag       = tag;
     req->recv.tag.tag_mask  = tag_mask;
-    req->recv.tag.cb        = cb;
+    req->recv.tag.cb        = param->cb.recv;
```

Comment:
what if CALLBACK flag is not set and `param->cb.recv` is just some garbadge?
would need to ensure we always check req->flags, but not the `req->recv.tag.cb` as indication of callback status

Reply:
in case if CALLBACK flag is not set, then cb value is not used & initialized by garbage, so, no affect will be here